unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionSceneAdminsSettingsController.cs

```diff
@@ -0,0 +1,224 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+internal class SectionSceneAdminsSettingsController : SectionBase, ISelectSceneListener, 
+                                                      ISectionUpdateSceneAdminsRequester, ISectionUpdateSceneBannedUsersRequester
+{
+    public const string VIEW_PREFAB_PATH = "BuilderProjectsPanelMenuSections/SectionSceneAdminsSettingsView";
+    
+    public event Action<string, SceneAdminsUpdatePayload> OnRequestUpdateSceneAdmins;
+    public event Action<string, SceneBannedUsersUpdatePayload> OnRequestUpdateSceneBannedUsers;
+
+    private readonly SectionSceneAdminsSettingsView view;
+    private readonly FriendsSearchPromptController friendsSearchPromptController;
+    private readonly UsersSearchPromptController usersSearchPromptController;
+    private readonly UserProfileFetcher profileFetcher = new UserProfileFetcher();
+    
+    private readonly SceneAdminsUpdatePayload adminsUpdatePayload = new SceneAdminsUpdatePayload();
+    private readonly SceneBannedUsersUpdatePayload bannedUsersUpdatePayload = new SceneBannedUsersUpdatePayload();
+
+    private List<string> admins = new List<string>();
+    private List<string> bannedUsers = new List<string>();
+    private string sceneId;
+
+    public SectionSceneAdminsSettingsController() : this(
+        Object.Instantiate(Resources.Load<SectionSceneAdminsSettingsView>(VIEW_PREFAB_PATH)),
+        FriendsController.i
+    )
+    {
+    }
+    
+    public SectionSceneAdminsSettingsController(SectionSceneAdminsSettingsView view, IFriendsController friendsController)
```

Comment:
I would create an interface for `SectionSceneAdminsSettingsView` and use it here in order to make the dependency of the view mockeable and be able to create tests for `SectionSceneAdminsSettingsController` (and not only for `SectionSceneAdminsSettingsView`).